dat/missions/neutral/pirbounty_dead.lua

```diff
@@ -320,7 +332,7 @@ function hunter_hail( arg )
 
    local text = share_text[ rnd.rnd( 1, #share_text ) ]
    tk.msg( share_title, text:format( name ) )
-
+   misn.appendLog("Payment of " .. math.floor(credits) .. "made for helping to kill the pirate")
```

Comment:
Assuming the log is supposed to be a useful reference for important past events, this is just cluttering it up with unnecessary information.